lib/opentoken-lib.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+
+/**
+ * @typedef opentokenLib
+ * @function createSignedTokenRequests
+ * @function createTokenRequest
+ */
+
+
+/**
+ * @typedef signedTokenRequests
+ * @property {SignedTokenRequest} private
+ * @property {SignedTokenRequest} public
+ */
+
+
+/**
+ * Creates an instance of opentokenLib.
+ *
+ * @param {string} host
+ * @param {string} logLevel
+ * @return {opentokenLib}
+ */
+module.exports = (host, logLevel) => {
+    var container, SignedTokenRequest, TokenRequest;
+
+    container = require("./container")(host, logLevel);
+    SignedTokenRequest = container.resolve("SignedTokenRequest");
+    TokenRequest = container.resolve("TokenRequest");
```

Comment:
Hmm interesting. I think I prefer it the way it is. If I had more then two calls I would be on your side with this. I am open to changing it though if you have a strong preference to it.